runtime/mainnet/src/benchmarks.rs

```diff
@@ -42,3 +67,183 @@ frame_benchmarking::define_benchmarks!(
 	[pallet_nfts, Nfts]
 	[pallet_assets, TrustBackedAssets]
 );
+
+parameter_types! {
+	pub ExistentialDepositAsset: Option<Asset> = Some((
+		RelayLocation::get(),
+		ExistentialDeposit::get()
+	).into());
+	pub const AssetHubParaId: ParaId = ParaId::new(1000);
+}
+
+impl pallet_xcm::benchmarking::Config for Runtime {
+	type DeliveryHelper = (
+		polkadot_runtime_common::xcm_sender::ToParachainDeliveryHelper<
+			XcmConfig,
+			ExistentialDepositAsset,
+			PriceForSiblingDelivery,
+			AssetHubParaId,
+			ParachainSystem,
+		>,
+	);
+
+	fn reachable_dest() -> Option<Location> {
+		Some(Location::parent())
+	}
+
+	fn teleportable_asset_and_dest() -> Option<(Asset, Location)> {
+		// No assets can be teleported.
+		None
+	}
+
+	fn reserve_transferable_asset_and_dest() -> Option<(Asset, Location)> {
+		ParachainSystem::open_outbound_hrmp_channel_for_benchmarks_or_tests(AssetHubParaId::get());
+
+		let who = frame_benchmarking::whitelisted_caller();
+		// Give some multiple of the existential deposit.
+		let balance = ExistentialDeposit::get() * 10_000;
+		let _ =
+			<Balances as frame_support::traits::Currency<_>>::make_free_balance_be(&who, balance);
+		let ah_on_pop: AccountId = LocationToAccountId::convert_location(&AssetHub::get())?;
+		let _ = <Balances as frame_support::traits::Currency<_>>::make_free_balance_be(
+			&ah_on_pop, balance,
+		);
```

Comment:
Funding AH sovereign account on pop because the benchmark tries to [withdraw from destination](https://github.com/paritytech/polkadot-sdk/blob/4a400dc1866f11707331fb6408df1055d0f42a70/polkadot/xcm/pallet-xcm/src/benchmarking.rs#L254) as a way to verify the benchmark.

I believe that makes sense for reserve transfers happening within the reserve itself. Not quite that for our case.